.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: python
```

Comment:
Hmm we've migrated off of travis; but it seems this is adding it back in. Why?

Reply:
I was unaware of this migration effort.  Can you describe the process of:

- Creating a README.md badge to link to the passing master build?
- How to get the URL of a triggered pull request (PR) build to the CloudBuild output?

These are critical items to nail down prior to a migration to reduce costly human toil.  Without a tight feedback loop here, I would ask that you not consider yourself migrated to Google CloudBuild.

Reply:
I think @mozafrank is the expert here

Reply:
README badge doesn't exist without going through a lot of hoops, there isn't a travis-like one at this time. When I looked into this about 2 weeks ago the value of the badge would require a pubsub and function. 

The URL should exist in the trigger check: it provides a link to investigate the build further upon failures.

Reply:
If you have issues viewing the link it provides:
![Screen Shot 2020-06-16 at 12 56 45 PM](https://user-images.githubusercontent.com/53873986/84822025-e6e44780-afd0-11ea-8d3b-37a4ea6d1644.png)

This is the link it's in the bottom of the status check/image.

https://console.cloud.google.com/cloud-build/builds/1fb12b06-7064-4d54-8657-677608c9c0dd?project=8078024812

Reply:
---------------------------------------------------------------------------------------------------------------------------------------------------
TOTAL                                                                                                                 40269  23756    41%
ERROR: InvocationError for command /workspace/.tox/py37/bin/coverage report -m '--omit=*/test*' --show-missing --fail-under=45 (exited with code 2)
___________________________________ summary ____________________________________
ERROR:   py37: commands failed
The command '/bin/sh -c tox' returned a non-zero code: 1
ERROR
ERROR: build step 0 "docker" failed: step exited with non-zero status: 1
---------------------------------------------------------------------------------------------------------------------------------------------------

I think these lines in the build are what you're looking for

Reply:
I will gladly remove the `.travis.yml` file but do so under duress of not being able to answer common questions about the state of a master branch build easily.

Reply:
I don't see a downside to additionally using travis if it adds value (and we're already paying for the travis builds), and we can continue to have the CICD discussion.

Reply:
The only thing is to ensure that we aren't doing one form of tests in one cicd and then another form in the other cicd; the goal would be to on a single cicd system; both builds should fail or succeed in the same fashion if we are adding Travis, the build should fail when CBuild fails, and vice versa. 

--

If the only value add for travis is the badge, and it's deemed as enough of a value add, go for it. 

Whatever the case, we'll remove it when/if the same functionality is available for CBuild.

Reply:
`.travis.yml` removed after badge addition.